### ISSUE TITLE
Route-Based VPN tunnels

### ISSUE THREAD

**Issue by EtienneMILON** [Forcepoint/fp-NGFW-SMC-python#7]
Hello,

I am able to configure tunnels in Policy-Based VPN with PolicyVPN > tunnels > endpoint_tunnels:
```
pb_vpn = PolicyVPN("my_pb_vpn")
for tunnel in pb_vpn.tunnels:
     for endpoint_tunnel in tunnel.endpoint_tunnels:
          print(endpoint_tunnel)
```
I can't find the same for the Route-Base VPN:
```
rb_vpn = RouteVPN("my_rb_vpn")
rb_vpn.tunnels
>>> AttributeError: <class 'smc.vpn.route.RouteVPN'> object has no attribute 'tunnels'
```
Is there a way to access it?

Best regards,
Etienne

**Comment by ggrimaux** [comment]
Hello,

Policy VPN and Route-Based VPN are very different. 
I did not tried it but maybe you should check in
[smc-python Route-Based VPN doc](https://smc-python.readthedocs.io/en/latest/pages/reference.html?highlight=route-based%20vpn#module-smc.vpn.route)

BR,
/Greg.

**Comment by thomasdevulder** [comment]
Could access to endpoints each side of rbvpn tunnel:

Example in Demo mode:

print(list(RouteVPN.objects.all()))

rb_vpn = RouteVPN("GRE-Plano VPN Gateway-Santa Clara VPN Gateway")
print("local endpoint (rbvpn_tunnel_side_a)"+str(rb_vpn.local_endpoint.endpoint)+" "+str(rb_vpn.local_endpoint.tunnel_interface))
print("remote endpoint (rbvpn_tunnel_side_b)"+str(rb_vpn.remote_endpoint.endpoint)+" "+str(rb_vpn.remote_endpoint.tunnel_interface))

=>
local endpoint (rbvpn_tunnel_side_a)InternalEndpoint(name=172.31.14.254) TunnelInterface(name=Tunnel Interface 1401)
remote endpoint (rbvpn_tunnel_side_b)InternalEndpoint(name=172.31.15.254) TunnelInterface(name=Tunnel Interface 1500)

Hope it could help.

**Comment by thomasdevulder** [comment]
Asked Alex to help, he knows well VPN

**Comment by EtienneMILON** [comment]
Hi Thomas,

Thanks for the answer.
What are the SMC and smc-python minimum versions?
I use SMC 6.5 and fp-NGFW-SMC-python 0.7.0b26, I have well access to "rb_vpn.local_endpoint.tunnel_interface" and "rb_vpn.remote_endpoint.tunnel_interface" but "rb_vpn.local_endpoint.endpoint" and "rb_vpn.remote_endpoint.endpoint" give nothing.

Best regards,
Etienne

**Comment by fourticq** [comment]
Hi @EtienneMILON ,

Indeed the feature was missing for Route-Based VPN tunnels.
It has now been implemented and available since 0.7.0b27 (see details [here](https://github.com/Forcepoint/fp-NGFW-SMC-python/blob/master/smc/vpn/route.py)).

You can now iterate through tunnels for Route-Based VPN (having VPN Type) using **rb_vpn.tunnels** method.
